rasa_original.py

```diff
@@ -0,0 +1,82 @@
+import aiohttp
+import asyncio
+import async_timeout
+from flask import Flask, request, abort
+
+
+loop = asyncio.new_event_loop()
+app = Flask(__name__)
+
+
+async def incoming_2_outgoing_payload(payload):
+    """Conveert json to xml."""
+    from decoder import Dict2XML
+    # need to convert int keys to str, dict2xml can't handle int values
+    return Dict2XML().parse({
+        "vacation": {k: str(v) for k, v in payload.items()}})
```

Comment:
Library doesn't seem to work , maybe its deprecated. 

```suggestion
    from dicttoxml import dicttoxml
    # need to convert int keys to str, dict2xml can't handle int values
    return dicttoxml({
        "vacation": {k: str(v) for k, v in payload.items()}})
```